dynawo/sources/Common/Dictionaries/DYNError_en_GB.dic

```diff
@@ -207,6 +207,8 @@ NumericalErrorFunction      =             A numerical error occurs during the ca
 //---CSTR--------------------
 //---DYD---------------------
 ModelIDNotUnique            =             model id (%1%) is not unique
+UnitModelIDSameAsModelName  =             model %1% has the same name as the parent modelica model
```

Comment:
I would rather use a vocabulary closer to the xml terminology in order to clearly understand what is the parent modelica model, the modelica class, the model name etc. I suggest the following error messages:
- unitDynamicModel %1%  id should be different from modelicaModel id
- unitDynamicModel %1% id should be different from its name

What do you think?

Reply:
Sounds good!

Reply:
Solved